modules/messages/client/controllers/thread.client.controller.js

```diff
@@ -32,6 +32,12 @@
         messageIdsInView = [],
         editorContentChangedTimeout;
 
+    // Vars for Host Module
+    var fromDateDisp,
```

Comment:
Ah ! Sure I was putting comments for functions need to put more here, I think I will put more in detail in thread dimension directive & other little places too !